desmume/src/frontend/posix/meson.build

```diff
@@ -203,6 +205,10 @@ endif
 if dep_agg.found()
   dependencies += dep_agg
   add_global_arguments('-DHAVE_LIBAGG', language: ['c', 'cpp'])
+  if get_option('frontend-gtk')
+    add_global_arguments('-DAGG2D_USE_VECTORFONTS', language: ['c', 'cpp'])
```

Comment:
you add this unconditionally, so i wonder if it still compiles if not set. also how does it look like if not set ?

Reply:
Maybe it's better to add meson option to use vector fonts then?

Reply:
i'd enable it unconditionally if fontconfig lib is found

Reply:
> i'd enable it unconditionally if fontconfig lib is found

Done